main.py

```diff
@@ -46,16 +46,43 @@ def is_out_of_bounds(self):
 class ScoreCard:
     def __init__(self):
         self.position = [3, 3]
+        self.highscorepos = [3,20]
         self.score = 0
         self.font = pygame.font.Font('freesansbold.ttf', 20)
-
+        self.load_highscore()
+
+    def __del__(self): self.save_highscore()
+    
+    def load_highscore(self):
+        #load high score
+        try:
+            with open('score/highscore.txt', 'r+') as f:
+                self.highscore = int(f.read())
+        except:
+            with open('score/highscore.txt', 'w') as f:
+                self.highscore = 0
```

Comment:
I guess you forgot to write high score as 0 in the file as you are opening in write mode.